server/db/matters.js

```diff
@@ -163,7 +179,7 @@ function addNewMatter (matter, db = knex) {
   return db('matters')
     .insert({
       // 'id as referenceNumber',
-      'category': matter.category,
+      // 'category_id': matter.category,
       'subcategories': matter.subcategories,
```

Comment:
I don't think we will need category and subcategories referenced inside matter? I think it should just be in the join table, and if we want to know which category/categories a matter falls under then we have to join the matters table with the categories table across the matters_categories join table.

If we reference it inside the matters table then it can only be connected to one

Reply:
On saying that though, if we are going to say that it can only have one category then we can reference it in the matters table. Subcategories will definitely have to be over a join table though